docs/data-structures/Circular-Linked-list.mdx

```diff
@@ -0,0 +1,457 @@
+        usage : remove(value)
+        '''
+        node = self.head
+        while node:
+            if node.next.value == data:
+                nextNode = node.next.next
+                node.next = nextNode
+                self.size = self.size-1
+                break
+            else:
+                node = node.next
+
+        else:
+            print("Value not Found")
+
+        print("Removing",data,"from the list")
+        
+        self.printAll()
+
+
+# Class for creation of nodes in Linked List
+class Node:
+    def __init__(self,data):
+        self.value = data
+        self.next = None
+
+if __name__ == '__main__':
+
+    # Taking Input from User
+    user_input = list(map(int,input("Enter initial values to be added in List \n").split()))
+
+    # Setting First Value as HEAD
+    l  = circular_llist(Node(user_input[0]))
+
+    # Creation of the list
+    for inputs in range(1,len(user_input)):
+        l.create(Node(user_input[inputs]))
+
+    print('''
+     PRESS  :  TO 
+     ----------------------
+       1    :  APPEND value at the END of list.
+       2    :  PREPEND value at the START of the list.
+       3    :  INSERT value at the desired Index
+       4    :  DELETE value from the list
+       5    :  PRINT ALL the values of your list
+       6    :  GET SIZE of your List
+       7    :  QUIT The program
+    -----------------------------
+    ''')
+
+
+    choice = 1
+    while choice!=7:
+        print("Enter Your Choice")
+        choice = int(input())
+        if choice==1:
+            user_value = int(input("Enter the value you want to APPEND\n"))
+            l.append(Node(user_value))
+        elif choice==2:
+            user_value = int(input("Enter the value you want to PREPEND\n"))
+            l.prepend(Node(user_value))
+        elif choice==3:
+            user_value = int(input("Enter the value you want to INSERT\n"))
+            user_desired_index = int(input(f"Enter the index for {user_value}\n"))
+            l.insert(Node(user_value),user_desired_index)
+        elif choice==4:
+            user_value = int(input("Enter the value to be DELETED\n"))
+            l.remove(user_value)
+        elif choice==5:
+            l.printAll()
+        elif choice==6:
+            l.getSize()
+        elif choice!=7:
+            print("Invalid choice. Try Again!")
+        
+    else:
+        print("ThankYou! for using our LinkedList\nHAve a Nice Day")
+
+```
+</TabItem>
+</Tabs>
+
+## Sample Input & Output
+
+#### Sample Input
+
+```
+Insertion in circular linkedlist
+enter the number of elements
+6
+enter element number 0-
+4
+enter element number 1-
+7
+enter element number 2-
+2
+enter element number 3-
+9
+enter element number 4-
+3
+enter element number 5-
+1
+Elements inserted
+1 -> 3 -> 9 -> 2 -> 7 -> 4
+enter the number of elements you want to delete
+2
+enter position of number 
+1
+enter position of number 
+3
+```
+
+#### Sample Output
+
+```
+1 -> 3 -> 7 -> 4
+```
```

Comment:
The I/O sample of C++ and Python doesn't seem to match. Can you fix?

Reply:
> The I/O sample of C++ and Python doesn't seem to match. Can you fix?

I could make then exactly the same, but I changed some parts. Now they're somewhat same.